tests/driver_hdc1000/main.c

```diff
@@ -55,14 +55,13 @@ int main(void)
             puts("Start measure failed.");
             return -1;
         }
-        vtimer_usleep(HDC1000_CONVERSION_TIME);
```

Comment:
I think the original `vtimer_usleep(HDC1000_CONVERSION_TIME);` was more educational (since there is no other example of using the hdc1000 device, the test case can be seen as a usage example)